src/components/LoginForm/LoginForm.tsx

```diff
@@ -0,0 +1,51 @@
+import ButtonLink from '../common/ButtonLink/ButtonLink';
+
+import './LoginForm.scss';
+
+// homepage - okno logowania
+
+interface Props {
+
+}
+
+
+const LoginForm: React.FC<Props> = () => {
+
+
+    return (
+        <form className='form-login'>
+            <img className='form-login__logo' src={require('../../images/logo-mk.png')} alt='' width='124' height='76' />
+            <label className='form-login__label'>
+                <input className='form-login__input'
+                    type='text'
+                    id='email'
+                    placeholder='E-mail'>
+                </input>  
+            </label>
+                          
+            <label className='form-login__label'>
+                <input className='form-login__input'
+                    type='text'
+                    id='password'
+                    placeholder='Hasło'>
+                </input>
+            </label>
+            
+            <div className='form-login__div'>
+                <div className='form-login__login'>            
+                    <ButtonLink customClass='form-login__button' type='submit' label='Zaloguj się'/>
+                    <p>
+                        <ButtonLink target='#' label='Zapomniałeś hasła?'/>
+                    </p>
+                </div>
+                <div className='form-login__register'>
```

Comment:
The registration field can be removed. Cuba on Live said it does not have to be. 
It will be a platform, to get on it you need to know Admin :) 
Admin creates accounts for students and HR. The student or HR gets an email with an activation link.
App for insiders :)